scripts/install.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
I'm not familiar with Docker but do these files need to be named `script/install.sh` and `script/run.sh`?  I would prefer they be put in a directory just for Docker things so it's clear what they're for.  Also, these will need license and copyright headers like the other files.